src/ratingservice/main.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from flask import Flask
+import psycopg2
+
+db_user = os.environ.get('CLOUD_SQL_USERNAME')
+db_password = os.environ.get('CLOUD_SQL_PASSWORD')
+db_name = os.environ.get('CLOUD_SQL_DATABASE_NAME')
+db_connection_name = os.environ.get('CLOUD_SQL_CONNECTION_NAME')
```

Comment:
Do you fail gracefully if any environment variables are unset?

Reply:
It won't fail gracefully because the server can't proceed without db connection strings.

Reply:
What does the error look like though? Will it be sensible to userss? It's usually a good practice to check the variable values and exit(1) with a clear message if they don't look right

Reply:
To users, it will be 500 server internal error since the server hasn't started it yet. The error is auto reported to the Error Reporting.